engines/director/castmember/filmloop.cpp

```diff
@@ -88,7 +89,7 @@ Common::Array<Channel> *FilmLoopCastMember::getSubChannels(Common::Rect &bbox, C
 		int16 width = src._width * widgetRect.width() / _initialRect.width();
 		int16 height = src._height * widgetRect.height() / _initialRect.height();
 
-		Channel chan(&src);
+		Channel chan(nullptr, &src);
```

Comment:
please add here a comment explaining why Score is `nullptr` here.

Reply:
Done